db/schema.ts

```diff
@@ -11,6 +11,7 @@ export const challenges = sqliteTable("challenges", {
   metadata: text("metadata", { mode: "json" }),
   createdAt: text("created_at").default(sql`CURRENT_TIMESTAMP`),
   updatedAt: text("updated_at").default(sql`CURRENT_TIMESTAMP`),
+  deleted: integer("deleted").notNull().default(0),
```

Comment:
SQLite does not have native booleans, but drizzle does let us interpret them as booleans with the following

```suggestion
deleted: integer("deleted", { mode: 'boolean' }).notNull().default(0),
```

See the docs for more info: https://orm.drizzle.team/docs/column-types/sqlite#boolean